### ISSUE TITLE
Command Quick Reference: nvda+k missing

### ISSUE THREAD

**Issue by hwf1324** [nvaccess/nvda#15791]
Due to the deliberate omission of certain parts of the quick start guide when creating the command quick reference, a new script that were only added in the quick start guide were not included

For example, `NVDA+k` is missing from the quick start guide

**Comment by CyrilleB79** [comment]
Have you found other missing gestures? Which ones?

**Comment by hwf1324** [comment]
No, I didn't double check that there were no other gestures.
